packages/main/src/themes/base/BusyIndicator-parameters.css

```diff
@@ -1,5 +1,5 @@
 :root {
 	--_ui5_busy_indicator_color: var(--sapContent_IconColor);
 	--_ui5_busy_indicator_focus_outline:  var(--sapContent_FocusWidth) var(--sapContent_FocusStyle) var(--sapContent_FocusColor);
-	--_ui5_busy_indicator_block_layer: color-mix(in oklch, transparent, var(--sapBlockLayer_Background) 20%);
-}
\ No newline at end of file
+	--_ui5_busy_indicator_block_layer: color-mix(in oklch, transparent, var(--sapBackgroundColor) calc((1 - var(--sapContent_DisabledOpacity)) * 100%));
```

Comment:
Any idea why we use the Oklab color space here? I am not sure if it will give the expected results all the time